CHANGELOG.md

```diff
@@ -1,4 +1,27 @@
 # Changelog
+## AMI Release v20210830
```

Comment:
looks like the release version in title is not correct

Reply:
Right. Let me resolve that.